.github/workflows/benchmarks.yml

```
@@ -11,6 +11,7 @@ on:
 
 jobs:
   benchmark:
+    if: false
```

actionlint (1.7.12)

[error] constant expression "false" in condition. remove the if: section

(if-cond)